source/extensions/filters/network/rbac/rbac_filter.cc

```diff
@@ -10,6 +10,10 @@ namespace Extensions {
 namespace NetworkFilters {
 namespace RBACFilter {
 
+static const std::string engine_result_allowed = "allowed";
+static const std::string engine_result_denied = "denied";
+static const std::string shadow_engine_result_field = "shadow_engine_result";
```

Comment:
@liminw @quanjielin 
Can we merge this with the HTTP filter to use the same set of constants? Instead of two sets of constants representing essentially the same thing.

Reply:
we should make tcp/http aligned, the work could happen in this PR or separated PR.

Reply:
Unsure if renaming the HTTP metadata fields/values will cause any regression externally.

Reply:
From istio side, it's marked as Alpha feature, which is 'No guarantees on backward compatibility'.

Reply:
Tracked via #4981.